docs/reference/ImageGrab.rst

```diff
@@ -20,7 +20,9 @@ or the clipboard to a PIL image memory.
     .. versionadded:: 1.1.3 (Windows), 3.0.0 (macOS)
 
     :param bbox: What region to copy. Default is the entire screen.
+                 Note that on Windows OS, the top-left point may be negative if ``all_screens=True`` is used.
     :param include_layered_windows: Includes layered windows. Windows OS only.
+    :param all_screens: Capture all monitors. Windows OS only.
```

Comment:
Please could you a `.. versionadded:: 7.0.0` for this param?

Compare with:

* https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.PIL.ImageDraw.ImageDraw.line

* https://github.com/python-pillow/Pillow/blob/master/src/PIL/ImageDraw.py

Reply:
@hugovk to be clear, do you mean 7.0, the end of year release, or 6.2, the next one?

Reply:
Oops, I meant 6.2.0, thanks for catching that!

Reply:
Done. I also added it for `include_layered_windows` (6.1.0).